src/Tizen.NUI.Components/Controls/Toast.cs

```diff
@@ -283,7 +294,7 @@ public uint Duration
         /// Apply style to toast.
         /// </summary>
         /// <param name="viewStyle">The style to apply.</param>
-		/// <since_tizen> 8 </since_tizen>
+		[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
could you check indentation? you can use "ctrl + k + d" in VS. ^^;

Reply:
Done! Thank you :)